test/run_cli.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const ChildProcess = require('child_process');
+const Path = require('path');
+
+const labPath = Path.join(__dirname, '..', 'bin', '_lab');
+
+module.exports = (args, callback) => {
+
+    return new Promise((resolve, reject) => {
+
+        const cli = ChildProcess.spawn('node', [labPath, ...args]);
+        let output = '';
```

Comment:
FWIW it's faster to push in arrays and join at the end, not sure it'll dramatically change test suite time though ;)